--- a/hairtunes.c	2013-01-21 14:48:30.000000000 +0800
+++ b/hairtunes.c	2013-02-18 14:01:39.027013480 +0800
@@ -447,13 +447,7 @@ static void *rtp_thread_func(void *arg)
             if (plen >= 16) {
                 buffer_put_packet(seqno, pktp, plen);
             } else {
-                // resync?
-                if (type == 0x56 && seqno == 0) {
-                    fprintf(stderr, "Suspected resync request packet received. Initiating resync.\n");
-                    pthread_mutex_lock(&ab_mutex);
-                    ab_resync();
-                    pthread_mutex_unlock(&ab_mutex);
-                }
+                continue;
             }
         }
     }
