--- a/squeezelite.h
+++ b/squeezelite.h
@@ -293,6 +293,19 @@ typedef int sockfd;
 
 #if EVENTFD
 #include <sys/eventfd.h>
+
+#if defined(__UCLIBC__)
+static int eventfd_read(int fd, eventfd_t* value)
+{
+	return (read(fd, value, sizeof(*value)) == sizeof(*value)) ? 0 : -1;
+}
+
+static int eventfd_write(int fd, eventfd_t value)
+{
+	return (write(fd, &value, sizeof(value)) == sizeof(value)) ? 0 : -1;
+}
+#endif
+
 #define event_event int
 #define event_handle struct pollfd
 #define wake_create(e) e = eventfd(0, 0)
